pom.xml

```diff
@@ -334,22 +339,6 @@
 					<artifactId>maven-source-plugin</artifactId>
 					<version>3.0.1</version>
 				</plugin>
-				<plugin>
```

Comment:
Does this mean we don't use this plugin at all?

Reply:
It's already in a build pom and these re-definitions just cause downgrades of versions.